src/test/java/posters/pageobjects/pages/user/DeleteAccountPage.java

```diff
@@ -29,9 +29,10 @@
 
     @Override
     @Step("ensure this is a delete account page")
-    public void isExpectedPage()
+    public DeleteAccountPage isExpectedPage()
     {
         deleteForm.should(exist);
```

Comment:
Please call `super.isExpectedPage();` at first in case we would add some default functionality.

Reply:
@ufjena this hasn't been covered yet.